cns/restserver/restserver.go

```diff
@@ -49,6 +49,10 @@ type wireserverProxy interface {
 	UnpublishNC(ctx context.Context, ncParams cns.NetworkContainerParameters, payload []byte) (*http.Response, error)
 }
 
+type IMDSClient interface {
```

Comment:
nit but this probably doesn't need to be publicly exported

Reply:
correct. Fixed.